extensions/email/src/main/java/org/mule/extension/email/internal/exception/EmailException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal.exception;
+
+/**
+ * Is the base {@link RuntimeException} type for the Email Connector any other exceptions
+ * thrown by the connector code will use or be based on this exception.
+ *
+ * @since 4.0
+ */
+public class EmailException extends RuntimeException
```

Comment:
also doesn't seem to add value. In either case should extend MuleRuntimeException but why not just use that class?